src/pal/prebuilt/corerror/mscorurt.rc

```diff
@@ -126,8 +126,8 @@ BEGIN
 	MSG_FOR_URT_HR(CORDBG_E_STATIC_VAR_NOT_AVAILABLE) "A static variable is not available because it has not been initialized yet."
 	MSG_FOR_URT_HR(CORDBG_E_CANT_SETIP_INTO_OR_OUT_OF_FILTER) "SetIP cannot leave or enter a filter."
 	MSG_FOR_URT_HR(CORDBG_E_CANT_CHANGE_JIT_SETTING_FOR_ZAP_MODULE) "JIT settings for ZAP modules cannot be changed."
-	MSG_FOR_URT_HR(CORDBG_E_CANT_SET_IP_OUT_OF_FINALLY_ON_WIN64) "SetIP is not possible because it would move EIP from within a finally clause to a point outside of one on WIN64 platforms."
-	MSG_FOR_URT_HR(CORDBG_E_CANT_SET_IP_OUT_OF_CATCH_ON_WIN64) "SetIP is not possible because it would move EIP from within a catch clause to a point outside of one on WIN64 platforms."
+	MSG_FOR_URT_HR(CORDBG_E_CANT_SET_IP_OUT_OF_FINALLY_ONBIT64) "SetIP is not possible because it would move EIP from within a finally clause to a point outside of one on WIN64 platforms."
```

Comment:
This should be `_BIT64`.

Reply:
Also, does the WIN64 in the message need to be updated?

Reply:
Unclear. I am not able to find where these errors are used.